code/modules/spells/aoe_turf/conjure/doppelganger.dm

```diff
@@ -16,6 +16,17 @@
 
 var/list/doppelgangers = list()
 
+// Sanity : don't copy more than one guy
+/spell/aoe_turf/conjure/cast_check(skipcharge = 0,mob/user = usr)
+	var/list/L = view(user, 0)
+	var/list/mob/moblist = list()
+	L -= user
+	for (var/mob/M in L)
+		moblist.Add(M)
+	if (moblist.len)
```

Comment:
I wanted to add an error message, but since `cast_check` is called twice in spellcode it would be displayed twice.